### PR TITLE
Display graphs on the request page

### PR DESCRIPTION
**Pull request by mscottford**
Adds XPlotly as a submodule and uses it to show the line graphs on the request page.

### CODE REVIEW

**Comment by mscottford** [comment]
Very much a work in progress, but we have a graph!

Screen Shot 2020-07-23 at 12 00 49 AM

**Comment by mscottford** [comment]
Updated screenshot showing multiple graphs.

screencapture-localhost-5000-AnalysisRequests-bff1f6a9-c4d8-4e85-82f6-5be129071c6c-2020-08-06-11_57_17

File: Freshli.Web/Controllers/AnalysisRequestsController.cs [hunk 1]
```diff
@@ -32,7 +36,158 @@ public class AnalysisRequestsController : Controller {
...
+          areaSeries.Add(new Scatter {
+            name = name,
+            x = projectDates,
+            y = projectLibYearValues
+          });
+        }
+
+        var chart = Chart.Plot(areaSeries.ToArray());
+        chart.WithLayout(new Layout.Layout {
+          title = "LibYear over time per dependency",
```
Comment (nickiemc) [inline Freshli.Web/Controllers/AnalysisRequestsController.cs:94]: Use title case to match other graph titles.

Comment (mscottford) [reply Freshli.Web/Controllers/AnalysisRequestsController.cs]: Fixed in e0988a1.

File: README.md [hunk 2]
```diff
@@ -22,7 +22,9 @@ For each language that the tool supports, the libyear metric is computed for eac
 
 ### Source Code Repositories
 
-* Git
+* [github.com/corgibytes/freshli](https://github.com/corgibytes/freshli)
+* [github.com/fslaborg/XPlot](https://github.com/fslaborg/XPlot)
+  * Freshli currently pulls in XPlot as a submodule as the current release does not include a critical bug fix
```
Comment (mscottford) [inline README.md]: This section was meant to list the source code repositories that were supported for history. I'm going to move the content that you added to somewhere else.

Comment (mscottford) [reply README.md]: You've actually got the important bits elsewhere, so I'm going to remove this and add more context here instead.

**Comment by mscottford** [comment]
Once this passes CI, I'm going to go ahead and merge it in.

File: README.md (hunk 2 above)
Comment (nickiemc) [reply README.md]: Ah, gotcha. I wasn't quite sure and assumed... 😬

**Pull request merged**